index.html

```diff
@@ -37,6 +37,148 @@ <h1>Hey there. I'm <br />Subrat </h1>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div>      
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div> 
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div> 
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div> 
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div>                         
+    </section>
+    <section id="aboutMself">
+      <h2 class="cardHeading">About Me</h2>
+      <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a href="#" class="cardAction">Get My Resume</a>
+      <ul class="lists">

```

Comment:
Kindly implement the button interaction for this also. You can refer to the pull request comment for this situation also.

Reply:
implemented